### PR TITLE
12035 - Mouse-over Stack Viewers allow configuring thickness of outer border, and more padding/stretching options

### PR DESCRIPTION
**Pull request by Cattlesquat**
(This one depends on #11964 so I'll make it fully active once that one merges) 

image

### CODE REVIEW

File: vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java [hunk 1]
```diff
@@ -469,37 +495,28 @@ protected void drawGraphics(Graphics g, @SuppressWarnings("unused") Point pt, JC
...
+        if (combineCounterSummary && stretchWidthSummary) {
+          drawLabel(g, new Point(lastPieceBounds.x - 1, y), pieces.isEmpty() ? terrainText : " ", LabelUtils.CENTER, LabelUtils.CENTER,
+            lastPieceBounds.width + 2,
+            lastPieceBounds.width + (int)((borderThickness * 2) * os_scale),
+            (int)((borderThickness - 1) * os_scale),
+            false,
+            (int)(((useInnerBottom ? borderInnerThickness : borderThickness) - 1) * os_scale),
+            (int)((borderThickness - 1) * os_scale)
+          );
+          didadukey = true;
```
Comment (uckelman) [inline vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java:513]: What's this?

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java:513]: (And just to clarify, this was already in the earlier PR, the one that dealt with the terrain, you're just noticing it now)

Comment (uckelman) [reply vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java:513]: Could you maybe name this something better? Nobody will have any idea what it is if anyone looks at it in the future.

Comment (Cattlesquat) [reply vassal-app/src/main/java/VASSAL/build/module/map/CounterDetailViewer.java]: Oh THAT kind of "what's this"... fixed up

**Pull request merged**